### PR TITLE
Version 0.1 -- Initial implementation

### PR DESCRIPTION
**Pull request by cberube**
To test:
- Review unit tests and ensure coverage
- Verify that the example in the README.md file is correct, and that log messages reach a Kinesis stream when using the example code

### CODE REVIEW

File: src/Handler/KinesisHandler.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    protected function write(array $record)
+    {
+        $content = $record['formatted'];
+        $content['StreamName'] = $this->streamName;
+
+        try {
+            /** @noinspection PhpUndefinedMethodInspection */
+            $this->kinesisClient->putRecord($content);
+        } catch (\Exception $ex) {
+            // We intentionally eat exceptions here -- the purpose of this handler is to emit logs to Kinesis for
```
Comment (nackjicholson) [inline src/Handler/KinesisHandler.php:58]: Worth explaining as a caveat in the README? Solicit feedback and PR's if people do want to use it that way?

**Comment by nackjicholson** [comment]
I've verified and reviewed the tests. From what I remember about monolog handlers this looks good. I haven't actually run it against a live kinesis stream, don't know if that's enough for an ok.

EDIT: Ran it, :ok: :shipit:

File: src/Handler/KinesisHandler.php (hunk 1 above)
Comment (cberube) [inline src/Handler/KinesisHandler.php:58]: Done.

File: src/Handler/KinesisHandler.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+    /**
+     * Constructor.
+     *
+     * @param KinesisClient $kinesisClient
+     * @param string $streamName
+     * @param int $level  The minimum logging level at which this handler will be triggered
+     * @param bool $bubble Whether the messages that are handled can bubble up the stack or not
+     */
+    public function __construct(KinesisClient $kinesisClient, $streamName, $level = Logger::CRITICAL, $bubble = true)
```
Comment (nackjicholson) [inline src/Handler/KinesisHandler.php]: Might should be Logger::INFO by default.

**Pull request merged**